Running in wrapper mode for '30'

Running python job.

[name: "CPU-QUOTA"
int_val: 80
int_val: 32
int_val: 29
int_val: 66
int_val: 89
int_val: 68
int_val: 41
, name: "DISK"
int_val: 21
int_val: 11
int_val: 11
int_val: 39
int_val: 18
int_val: 39
int_val: 33
, name: "MEMORY"
int_val: 22
int_val: 22
int_val: 33
int_val: 16
int_val: 14
int_val: 10
int_val: 15
, name: "NET"
int_val: 23
int_val: 16
int_val: 34
int_val: 5
int_val: 18
int_val: 13
int_val: 36
]
The paramters are {u'CPU-QUOTA': array([80, 32, 29, 66, 89, 68, 41]), u'NET': array([23, 16, 34,  5, 18, 13, 36]), u'DISK': array([21, 11, 11, 39, 18, 39, 33]), u'MEMORY': array([22, 22, 33, 16, 14, 10, 15])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
----------------------------------------
Problem running the job:
(<class 'paramiko.ssh_exception.BadAuthenticationType'>, BadAuthenticationType('Bad authentication type', [u'publickey']), <traceback object at 0x7fd41b7b08c0>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 147, in explore_spearmint
    client = re.get_client(machine)
  File "/home/ubuntu/throttlebot/src/remote_execution.py", line 6, in get_client
    client.connect(ip, username="quilt", password="")
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 424, in connect
    passphrase,
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 714, in _auth
    raise saved_exception
BadAuthenticationType: ('Bad authentication type', [u'publickey']) (allowed_types=[u'publickey'])
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 0.46 seconds.
